src/main/java/Car.java

```diff
@@ -0,0 +1,42 @@
+package RacingCar;
+
+import java.util.List;
+import java.util.Random;
+
+public class Car {
+    private static final int bound =10;
```

Comment:
상수는 대문자와 스네이크 케이스로 표현합니다. MY_NAME, HELLO_WORLD...
```suggestion
    private static final int BOUND = 10;
```